src/Publication/Publication.service.js

```diff
@@ -1,23 +1,27 @@
 import { Publication } from './Publication.model'
-import { Parcel } from '../Parcel'
-import { Estate } from '../Estate'
-import { ASSET_TYPES } from '../shared/asset'
+import { ASSETS } from '../Asset'
+import { PUBLICATION_ASSET_TYPES } from '../shared/publication'
 
 export class PublicationService {
   constructor() {
     this.Publication = Publication
-    this.Parcel = Parcel
-    this.Estate = Estate
   }
 
-  getModelFromAssetType(assetType) {
+  getPublicableAssetFromType(assetType) {
     if (!this.Publication.isValidAssetType(assetType)) {
       throw new Error(`Invalid publication asset_type "${assetType}"`)
     }
 
-    return {
-      [ASSET_TYPES.parcel]: this.Parcel,
-      [ASSET_TYPES.estate]: this.Estate
-    }[assetType]
+    return ASSETS[assetType]
+  }
+
+  getPublicableAssets(assetType) {
```

Comment:
it looks like the `assetType` argument is not used in this function

Reply:
👍